### ISSUE TITLE
weather_owm API Error

### ISSUE THREAD

**Issue by fxdgear** [ultrabug/py3status#1728]
Trying to figure out more information, but I'm not sure how to debug the built in modules. 

```
weather_owm {
    api_key = "<my_api_key>"
    location = (<my_lat>,<my_lon>)
}
```

Should be pretty simple, I've verified that I get a result using the api key and the latlon using curl. 
BUT like i said I'm not sure how to debug this module myself.

**Comment by fxdgear** [comment]
Ahh turns out the location needs to be a tuple of `(lon,lat)` not lat/lon as the docs describe. 

I was able to figure out how to set the debug logger, add a few print statements and found that on this line: https://github.com/ultrabug/py3status/blob/master/py3status/modules/weather_owm.py#L773-L774

we're fliping the lat longs.

Not sure what you wanna do, update the docs or update the module?

**Comment by lasers** [comment]
Aha! No wonder I never could get `location` to work okay on my side. I'd update the module to be consistent with latitude and longitude. Can you fix the `Saint-Denis` example too?

**Comment by lasers** [comment]
Closed via #1732.
